packages/@jsii/spec/lib/assembly.ts

```diff
@@ -795,6 +797,7 @@ export enum TypeKind {
   Class = 'class',
   Enum = 'enum',
   Interface = 'interface',
+  NamedUnion = 'union',
```

Comment:
Does this not need an assembly version bump? There will now be assemblies that old version of jsii have no idea what to do with.